src/main/java/supersymmetry/api/recipe/builders/SinteringRecipeBuilder.java

```diff
@@ -0,0 +1,46 @@
+package supersymmetry.api.recipe.builders;
+
+import gregtech.api.recipes.Recipe;
+import gregtech.api.recipes.RecipeBuilder;
+import gregtech.api.recipes.RecipeMap;
+import gregtech.api.util.EnumValidationResult;
+import gregtech.api.util.GTLog;
+import org.jetbrains.annotations.NotNull;
+import supersymmetry.api.recipe.properties.CoilingCoilTemperatureProperty;
+import supersymmetry.api.recipe.properties.SinterProperty;
+
+public class SinteringRecipeBuilder extends RecipeBuilder<SinteringRecipeBuilder> {
+
+    public SinteringRecipeBuilder() {
+
+    }
+
+    public SinteringRecipeBuilder(SinteringRecipeBuilder builder) {
+        super(builder);
+    }
+
+    @SuppressWarnings("unused")
+    public SinteringRecipeBuilder(Recipe recipe, RecipeMap<SinteringRecipeBuilder> recipeMap) {
+        super(recipe, recipeMap);
+    }
+
+    @Override
+    public SinteringRecipeBuilder copy() {
+        return new SinteringRecipeBuilder(this);
+    }
+
+    public SinteringRecipeBuilder usePlasma(boolean usePlasma) {
```

Comment:
You may want to add an additional method overload `usePlasma()`, to enable plasma without having to specify a boolean, for brevity.